lib/backend/dynamo/dynamodbbk.go

```diff
@@ -215,10 +215,10 @@ const (
 	hashKeyKey = "HashKey"
 )
 
-var (
+const (
 	// keyPrefix is a prefix that is added to every dynamodb key
 	// for backwards compatibility
-	keyPrefix = backend.KeyFromString("teleport")
+	keyPrefix = "teleport"
```

Comment:
Should we add a comment with warning here that updating keyPrefix may have side effects like described in the PR description?